CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## 1.17.3-dev
+* Enhancement - Distinct startContainerTimouts for windows/linux, introduce a new environment variable `ECS_CONTAINER_START_TIMEOUT`
```

Comment:
"introduce a new environment variable `ECS_CONTAINER_START_TIMEOUT` to make it configurable" - might be better 
nit: typo `startContainerTimeouts`

Reply:
OK